### PR TITLE
Avoid returning widgets lint improvement

### LINKED ISSUE #68: Improve avoid_returning_widgets to ignore override methods
We need to improve the avoid_returning_widgets rule to ignore all places where we are returning a Widget from the override methods instead of just ignoring methods with the `build` name.

### PR DESCRIPTION
**Pull request by Viktor-Mudrytskyi**
Closes #68

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.1.6
```
Comment (yurii-prykhodko-solid) [inline CHANGELOG.md:1]: Let's batch it with 0.1.5
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+- Improvement for avoid_returning_widget
```
Comment (yurii-prykhodko-solid) [inline CHANGELOG.md:3]: Let's be more specific
```suggestion
- Improvement for avoid_returning_widget fixing xxx
```

File: pubspec.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: solid_lints
 description:
   Lints for Dart and Flutter based on software industry standards and best
   practices.
-version: 0.1.5
+version: 0.1.6
```
Comment (yurii-prykhodko-solid) [inline pubspec.yaml:5]:
```suggestion
version: 0.1.5
```

**Review by yurii-prykhodko-solid — CHANGES_REQUESTED** [review]
Just a few minor tweaks unrelated to code.

**Review by yurii-prykhodko-solid — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 - Added `avoid_using_debug_print` rule
 - Fixed an issue with no_magic_number lint
+- Improvement for `avoid_returning_widget` lint:
+  - allows overrides
```
Comment (illia-romanenko) [inline CHANGELOG.md:6]:
```suggestion
  - ignores methods that override ones that return widget (build() for example)
```

**Review by illia-romanenko — APPROVED** [review]
LGTM - small naming suggestion on changelog. Let me know what you think.

**Pull request merged**